### PR TITLE
update Http Semantic Changes statement in changelog for instrumentation libraries (OTEL_SEMCONV_STABILITY_OPT_IN)

### PR DESCRIPTION
**Pull request by TimothyMothra**
Design discussion issue #4484
Follow up to #4662

## Changes
- Update Changelogs, using the same statement that @alanwest contributed in his PR.

## Merge requirement checklist

* [ ] [CONTRIBUTING](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/CONTRIBUTING.md) guidelines followed (nullable enabled, static analysis, etc.)
* [ ] Unit tests added/updated
* [ ] Appropriate `CHANGELOG.md` files updated for non-trivial changes
* [ ] Changes in public API reviewed (if applicable)

### CODE REVIEW

File: src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md [hunk 1]
```diff
@@ -2,10 +2,28 @@
...
+  `1.5.0-beta.1`.
+  * Note: this option will eventually be removed after the new HTTP and
+  network semantic conventions are marked stable. At which time this
+  instrumentation can receive a stable release, and the old HTTP and
+  network semantic conventions will no longer be supported. Refer to the
+  specification for more information regarding the new HTTP and network
+  semantic conventions for both
+  [spans](https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/database/database-spans.md)
+  and
+  [metrics](https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/database/database-metrics.md).
```
Comment (vishweshbankwar) [inline src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md:26]: Remove this - We don't have database metrics.

File: src/OpenTelemetry.Instrumentation.GrpcNetClient/CHANGELOG.md [hunk 2]
```diff
@@ -2,12 +2,29 @@
...
+  `1.5.0-beta.1`.
+  * Note: this option will eventually be removed after the new HTTP and
+  network semantic conventions are marked stable. At which time this
+  instrumentation can receive a stable release, and the old HTTP and
+  network semantic conventions will no longer be supported. Refer to the
+  specification for more information regarding the new HTTP and network
+  semantic conventions for both
+  [spans](https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/rpc/rpc-spans.md)
+  and
+  [metrics](https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/rpc/rpc-metrics.md).
```
Comment (vishweshbankwar) [inline src/OpenTelemetry.Instrumentation.GrpcNetClient/CHANGELOG.md:26]: same here, remove metrics.

File: src/OpenTelemetry.Instrumentation.GrpcNetClient/CHANGELOG.md [hunk 3]
```diff
@@ -2,12 +2,29 @@
 
 ## Unreleased
 
-* Updated Semantic Conventions to
-  [v1.21.0](https://github.com/open-telemetry/semantic-conventions/blob/v1.21.0/docs/rpc/rpc-spans.md).
-  This library can emit either old, new, or both attributes. Users can control
-which attributes are emitted by setting the environment variable
-OTEL_SEMCONV_STABILITY_OPT_IN.
-[#4658](https://github.com/open-telemetry/opentelemetry-dotnet/pull/4658)
+* The new HTTP and network semantic conventions can be opted in to by setting
```
Comment (vishweshbankwar) [inline src/OpenTelemetry.Instrumentation.GrpcNetClient/CHANGELOG.md:5]: same as https://github.com/open-telemetry/opentelemetry-dotnet/pull/4664/files#r1267130300

File: src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md [hunk 4]
```diff
@@ -2,10 +2,26 @@
...
+  * `http` - emit the new, frozen (proposed for stable) networking
+  attributes, and stop emitting the old experimental networking
+  attributes that the instrumentation emitted previously.
+  * `http/dup` - emit both the old and the frozen (proposed for stable)
+  networking attributes, allowing for a more seamless transition.
+  * The default behavior (in the absence of one of these values) is to continue
+  emitting the same network semantic conventions that were emitted in
+  `1.5.0-beta.1`.
+  * Note: this option will eventually be removed after the new
+  network semantic conventions are marked stable. At which time this
```
[the remark this answers is not in the document]
Comment (utpilla) [reply src/OpenTelemetry.Instrumentation.SqlClient/CHANGELOG.md:19]: Yes, we can skip this for Grpc and Sql instrumentation.

**Review by utpilla — APPROVED** [review]

**Pull request merged**